lib/native/mqtt.spec.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
👩‍⚖️ we can do this? I am so on board if so

Reply:
Yes. It's a soft push across the org, but I have a task in the Fix-It Friday backlog to delete years from our copyrights.